src/main/resources/palaemonTemplate.vm

```diff
@@ -73,6 +73,7 @@ services:
       #foreach($key in $INPUT_FILE_NAMES.keySet())
       $key: '$INPUT_FILE_NAMES.get($key)'
       #end
+      APP_PROVIDER_SECRET_0: '$env.get("APP_PROVIDER_SECRET_0")'
```

Comment:
`IEXEC_APP_PROVIDER_SECRET_0` (iExec prefix to reduce probability of collisions with app developer env vars)